eventing/samples/cronjob-source/README.md

```diff
@@ -68,11 +68,9 @@ kubectl apply -f cronjob-source.yaml
 We will verify that the message was sent to the Knative eventing system by
 looking at message dumper logs.
 
-1. Use [`kail`](https://github.com/boz/kail) to tail the logs of the subscriber.
-
-   ```shell
-   kail -l serving.knative.dev/service=message-dumper -c user-container --since=10m
-   ```
+```shell
+kubectl logs -l serving.knative.dev/service=message-dumper -c user-container
```

Comment:
How to enable `--since=10m` with `kubectl logs`?

Reply:
Just add the parameter `--since=10m` in commend `kubectl logs`. I update the commend here.